tests/installation/partitioning/separate_home.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The test module goes through the Suggested Partitioning wizard,
+# keeping all the default values but explicitly enables separate /home
+# partition.
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use strict;
+use warnings FATAL => 'all';
+use parent "installbasetest";
+
+sub run {
+    my $partitioner = $testapi::distri->get_partitioner();
+    $partitioner->edit_proposal(has_separate_home => 1);
```

Comment:
Perhaps in future when we have some declarative way to express the schedule we can think about passing modules some flags, because in this case seems clear that there is some code replication and the only difference is a the flag.
But it is a minor thing now, as it is much easy to understand now than before with the new OOP approach.

Reply:
With the current approach each test module expresses what it is doing (like 'partitioning/separate_home', 'partitioning/encrypt_lvm'). So, in the schedule we do not care about the data, we just call the module if it need to be called in the particular place and it will do all the things.

I also had thoughts about the approach when schedule might pass some data to the module and the module is just dumb. Like having 'separate_home.pm' and provide 1 or 0 from the schedule. It might decrease the amount of modules and also allow to operate with the test modules as with test steps. 

But I'm not sure if it will be implemented ever.